src/bosh-softlayer-cpi/softlayer/pool/softlayer_pool_creator.go

```diff
@@ -150,6 +150,8 @@ func (c *softLayerPoolCreator) createFromVMPool(agentID string, stemcell bslcste
 	return sl_vm_os, nil
 }
 
+func (c *softLayerPoolCreator) GetAgentOptions() AgentOptions { return c.agentOptions }
```

Comment:
@maximilien Do you think if it's feasible to change `VcapPassowrd` defined in `AgentOptions` to a private variable, then write a public function `GetVcapPassword` to retrieve the vcap password in `agent_options.go`? Thanks.